semconv/src/main/scala/org/typelevel/otel4s/semconv/resource/attributes/ResourceAttributes.scala

```diff
@@ -0,0 +1,981 @@
+        extends CloudPlatformValue("gcp_cloud_functions")
+
+    /** Google Cloud App Engine (GAE). */
+    case object GcpAppEngine extends CloudPlatformValue("gcp_app_engine")
+
+    /** Red Hat OpenShift on Google Cloud. */
+    case object GcpOpenshift extends CloudPlatformValue("gcp_openshift")
+
+    /** Red Hat OpenShift on IBM Cloud. */
+    case object IbmCloudOpenshift
+        extends CloudPlatformValue("ibm_cloud_openshift")
+
+    /** Tencent Cloud Cloud Virtual Machine (CVM). */
+    case object TencentCloudCvm extends CloudPlatformValue("tencent_cloud_cvm")
+
+    /** Tencent Cloud Elastic Kubernetes Service (EKS). */
+    case object TencentCloudEks extends CloudPlatformValue("tencent_cloud_eks")
+
+    /** Tencent Cloud Serverless Cloud Function (SCF). */
+    case object TencentCloudScf extends CloudPlatformValue("tencent_cloud_scf")
+
+  }
+
+  sealed abstract class AwsEcsLaunchtypeValue(value: String)
+  object AwsEcsLaunchtypeValue {
+
+    /** ec2. */
+    case object Ec2 extends AwsEcsLaunchtypeValue("ec2")
+
+    /** fargate. */
+    case object Fargate extends AwsEcsLaunchtypeValue("fargate")
+
+  }
+
+  sealed abstract class HostArchValue(value: String)
+  object HostArchValue {
+
+    /** AMD64. */
+    case object Amd64 extends HostArchValue("amd64")
+
+    /** ARM32. */
+    case object Arm32 extends HostArchValue("arm32")
+
+    /** ARM64. */
+    case object Arm64 extends HostArchValue("arm64")
+
+    /** Itanium. */
+    case object Ia64 extends HostArchValue("ia64")
+
+    /** 32-bit PowerPC. */
+    case object Ppc32 extends HostArchValue("ppc32")
+
+    /** 64-bit PowerPC. */
+    case object Ppc64 extends HostArchValue("ppc64")
+
+    /** IBM z/Architecture. */
+    case object S390x extends HostArchValue("s390x")
+
+    /** 32-bit x86. */
+    case object X86 extends HostArchValue("x86")
+
+  }
+
+  sealed abstract class OsTypeValue(value: String)
+  object OsTypeValue {
+
+    /** Microsoft Windows. */
+    case object Windows extends OsTypeValue("windows")
+
+    /** Linux. */
+    case object Linux extends OsTypeValue("linux")
+
+    /** Apple Darwin. */
+    case object Darwin extends OsTypeValue("darwin")
+
+    /** FreeBSD. */
+    case object Freebsd extends OsTypeValue("freebsd")
+
+    /** NetBSD. */
+    case object Netbsd extends OsTypeValue("netbsd")
+
+    /** OpenBSD. */
+    case object Openbsd extends OsTypeValue("openbsd")
+
+    /** DragonFly BSD. */
+    case object Dragonflybsd extends OsTypeValue("dragonflybsd")
+
+    /** HP-UX (Hewlett Packard Unix). */
+    case object Hpux extends OsTypeValue("hpux")
+
+    /** AIX (Advanced Interactive eXecutive). */
+    case object Aix extends OsTypeValue("aix")
+
+    /** SunOS, Oracle Solaris. */
+    case object Solaris extends OsTypeValue("solaris")
+
+    /** IBM z/OS. */
+    case object ZOs extends OsTypeValue("z_os")
+
+  }
+
+  sealed abstract class TelemetrySdkLanguageValue(value: String)
+  object TelemetrySdkLanguageValue {
```

Comment:
This doesn't include Scala 😂

Reply:
I have added one, but it appears that we will need to create a pull request in the specification repository once the SDK is ready

Reply:
This raises an important question: can these enums get more members in future releases? If that's the case, then we should not mark them as `sealed`, because it is not safe to do a total pattern match since there may be more `case`s added that will break that code.